### ISSUE TITLE
Create a separate call view that only has 1 box

### ISSUE THREAD

**Issue by askhogan** [blak3r/yaai#100]
In high volume call centers the stacking nature of the call popup can become annoying and intrusive.  Yet at the same time some users love the stacking nature because it acts as a reminder of calls to log notes to.  I have pondered on this issue for a long time.  I love the popup, but I hate getting complaints from users that the callpopup has stacked 10-12 times high because they are taking a high volume of calls and now they cant get their job done.  

There should be a duplicate view option created similar to Google Voice where there are two tabs in one call view.  One tab would have the active calls.  The other tab would have historical calls.

**Comment by blak3r** [comment]
Potentially easier solution would be to just add a user preference.  "Max Call Popups Windows" - only shows the X most recent calls.

Or, just give them a Close All button.

We definitely should make the X button easier to click.  I find I accidentally minimize when I want to close a lot.  You have to click on the text part which can be easy to miss.

**Comment by askhogan** [comment]
I think this the limitation on the call boxes is the simplest solution.  

The only reason why I thought the history approach may be better is that it allows people to see call history.  For whatever reason I have always liked the Google Voice approach of seeing a nice long list of all your past calls.  We could implement that without doing a full get_calls.   It could be a much lighter json call.

**Comment by blak3r** [comment]
Yeah I think we eventually should have a special colored box that would who
in it like two tabs.  One tab is your complete call history for say the day.
The other would be all your voicemails.  We wouldn't have to poll the
server until they click maximize.

I'll add that as a configuration parameter.  I got some other ones to add
as well.
